documentation/source/index.rst

```diff
@@ -39,6 +36,7 @@ Tutorials
 .. toctree::
    :maxdepth: 1
    :caption: Tutorials
+   :name: tutorials
```

Comment:
What's the purpose of this?

Reply:
It does link to this index page. The pages in the toctree are separate pages, so without ``:name:``, there wouldn't be a way to link to e.g. "all tutorials".